core/src/main/scala/scalacss/Attrs.scala

```diff
@@ -2126,7 +2156,18 @@ object Attrs {
    *
    * @see <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/transition-timing-function">MDN</a>
    */
-  final val transitionTimingFunction = Attr.real("transition-timing-function", Transform keys CanIUse.transitions)
+  object transitionTimingFunction extends TypedAttrBase {
+    override val attr: Attr = Attr.real("transition-timing-function", Transform keys CanIUse.transitions)
+    @inline def cubicBezier(x1: Double, y1: Double, x2: Double, y2: Double) = avl(new LT.CubicBezier(x1, y1, x2, y2))
+    @inline def steps(steps: Int, direction: LT.TimingFunctionDirection) = avl(new LT.Steps(steps, direction))
+    @inline def linear = avl(LT.linear)
+    @inline def ease = avl(LT.ease)
+    @inline def easeIn = avl(LT.easeIn)
+    @inline def easeInOut = avl(LT.easeInOut)
+    @inline def easeOut = avl(LT.easeOut)
+    @inline def stepStart = avl(LT.stepStart)
+    @inline def stepEnd = avl(LT.stepEnd)
```

Comment:
Here (and the groups above), please align all the whitespace so all the `=` are aligned. Big improvement of readablity. Does your IDE support multiple/verticle cursors? It's super quick with that feature.